src/main/java/io/github/jhipster/registry/config/LoggingConfiguration.java

```diff
@@ -25,63 +40,91 @@
 
     private final String serverPort;
 
-    private final String instanceId;
```

Comment:
This is strange, why remove instanceId ? This is needed for microservices.

Reply:
This is the code that generated from main generation. Just make sure it's identical with another microservices.

Reply:
You are right ! It seems it has been removed at some point 😢 
It used to be here https://github.com/jhipster/generator-jhipster/commit/7993651ec2e80c97c7d8cf3b9f636e0b0522eb1a
I need to put this back as is is essential for the console.

Reply:
So if that, you will need to update the generator to bring the instanceId back.

Reply:
Should we merge this request or waiting to the generator update?

Reply:
Agreed. I'm merging this. Thanks a lot for your time and congrats on your first contribution.
I changed your PR to point to the develop branch.

Reply:
Thank @PierreBesson, will do that in the next time.